Makefile

```diff
@@ -1,22 +1,42 @@
-.PHONY: clean image container container-smoke
+.PHONY: build clean image container container-smoke prune
 
 help:
 	@echo "  image        build the wellknown docker image"
 	@echo "  container    create a container from the wellknown image"
 	@echo "  container-smoke   smoke test the container"
+	@echo "  prune        clean up stray containers, images and references to them"
+
+PYDIR=src/grav
+DISTDIR=dist
+PYSRC=$(shell find $(PYDIR) -name '*.py')
+# TODO: make this handle versioning properly
+# if I planned on changing the version
+WHEEL=$(DISTDIR)/gravitational_audition-0.1.0-py3-none-any.whl
+PYTHON_SCHMUTZ=$(WHEEL)
 
 DOCKERDIR=wellknown-container
 IMAGE_ID=$(DOCKERDIR)/.image_id
 CONTAINER_ID=$(DOCKERDIR)/.container_id
+IMAGE_TAR=$(PYDIR)/wellknown.tar
+DOCKER_SCHMUTZ=$(IMAGE_TAR) $(IMAGE_ID) $(CONTAINER_ID)
```

Comment:
What's "schmutz"?

Reply:
It comes from Yiddish I believe, and generally means "mess" or "dirt".  E.g. you might say "Clean that schmutz off your face!" to a child sticky with ice cream.

I'm by no means attached to this particular phrasing if you feel something else is more understandable.